GoogleSignIn/Sources/GIDGoogleUser.m

```diff
@@ -95,6 +75,18 @@ - (nullable NSString *)openIDRealm {
   return grantedScopes;
 }
 
+- (GIDConfiguration *)configuration {
+  __block GIDConfiguration *configuration;
+  static dispatch_once_t onceToken;
```

Comment:
As with your previous use of the `static` storage type above, I don't think you want this variable to be shared by all instances of `GIDGoogleUser`.

Reply:
Fixed.